### DISCUSSION TITLE
Username label missing with SSO

### DISCUSSION THREAD

**Discussion by marshall89923** [argoproj/argo-workflows#13768] [Q&A]
https://github.com/dexidp/dex/discussions/3796

hello everyone,
I have configured dex config and ADFS as mentioned here:

https://github.com/dexidp/dex/issues/1309#issuecomment-428655923

and the SAML is working correctly. However in Argo Workflows, there are pipelines, and those pipelines are marked with the user executing them.
In these pipelines, I need both the username and email, for example, to perform Git operations.
Currently, the jobs are only marked with the email, and the username is missing.
How can I configure adfs and DEX Config to use username and password and not only email?

Thank you very much

**Comment by agilgur5** [comment]
Are you referring to Argo's [Workflow Creator](https://argo-workflows.readthedocs.io/en/release-3.5/workflow-creator/) feature that automatically labels certain Workflows?

`creator` is supposed to be [the SSO `subject`](https://github.com/argoproj/argo-workflows/blob/8905a0efee5549927d31178e93d97f0f528ff166/workflow/creator/creator.go#L18), which can be a username
